### DISCUSSION TITLE
convert raw file to ACEScg exr

### DISCUSSION THREAD

**Discussion by p0las** [OpenImageIO/oiio#4408] [Q&A]
Hi,
I'm struggling with this conversion and cannot find any documentation for the parameters that can be passed to libraw. Is there a documentation for that?

my current command is as follows:
`oiiotool --iconfig raw:ColorSpace ACES --iconfig raw:HighlightMode 1 -i "0H2A7632.CR2" --colorconvert ACES ACEScg -o "0H2A7632.exr"`
but it produces pink highlights. there is a libRaw argument `use_camera_wb` that might have some impact on this but I don't know what the equivalent one in oiiotool might be. Google is not very helpful and I totally failed to find any documentation for this.

**Comment by friendlyghostly** [comment]
I'm running into similar challenges. I have been using Rawline for this task but was hoping to move over to OIIO

**Comment by lgritz** [comment]
Pinging @antond-weta for advice

**Comment by antond-weta** [comment]
I understand that raw:HighlightMode=1 is the only mode which does not do highlight recovery, hence the pink highlights. You may want to try 0, or 2..9.

The raw plugin documentation can be found [here](https://openimageio.readthedocs.io/en/latest/builtinplugins.html#raw-digital-camera-files)
